### DISCUSSION TITLE
Ability to put pinned repos above the profile bio

### DISCUSSION THREAD

**Discussion by chase-lambert** [community/community#61784] [Other Feature Feedback, Questions, & Ideas]
I've gotten a little cutesy with my bio but I like it so want to keep it. Unfortunately it is a bit long and I would love to be able to put my pinned repos above that. Any chance you make our profile pages more customizable?

**Comment by moonfury-ops** [comment]
Thank you

**Comment by w8ste** [comment]
As far as i know this is not possible :/

**Comment by SaptarshiSarkar12** [comment]
Not Possible @chase-lambert.
